docs/rst/user_guide/atmosphere/intro.rst

```diff
@@ -0,0 +1,209 @@
+   :math:`k_{\mathrm{s}} = 10^{-3} \, \mathrm{km}^{-1}` and
+   :math:`k_{\mathrm{a}} = 10^{-5} \, \mathrm{km}^{-1}`
+   using:
+
+   .. code:: python
+
+      import eradiate
+      from eradiate import unit_registry as ureg
+      eradiate.set_mode("mono")
+
+      atmosphere = eradiate.scenes.atmosphere.HomogeneousAtmosphere(
+          toa_altitude = ureg.Quantity(120, "km"),
+          width = ureg.Quantity(1000, "km"),
+          sigma_s = ureg.Quantity(1e-3, "km^-1"),
+          sigma_a = ureg.Quantity(1e-5, "km^-1"),
+      )
+
+For more detail, refer to the guide page dedicated to
+:ref:`homogeneous atmospheres <sec-atmosphere-homogeneous>`.
+
+Heterogeneous atmosphere
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The heterogeneous atmosphere is characterised by non-uniform radiative
+properties.
+
+.. note::
+
+   At the moment, Eradiate provides 1D-heterogeneous atmosphere only.
+
+.. image:: fig/atmosphere-heterogeneous.png
+   :align: center
+   :scale: 50
+
+1D-heterogeneous atmospheres are characterised by radiative properties
+that vary with altitude.
+Together, the values of the radiative properties at different altitude points
+constitute the **radiative properties profile** of the atmosphere.
+The radiative properties profile is specified by the ``profile`` parameter of
+the :class:`~eradiate.scenes.atmosphere.HeterogeneousAtmosphere` class.
+You can either specify the radiative properties profile by choosing
+a profile from a list of registered profile types, or by providing
+:ref:`kernel volume data files <sec-atmosphere-heterogeneous-kernel_volume_data_files>`
+where the values of the extinction coefficient and albedo are written.
+
+.. note::
+
+   So far, only the
+   :class:`us76_approx <eradiate.radprops.rad_profile.US76ApproxRadProfile>`
+   radiative properties profile is available.
+   We are working on adding radiative properties profiles corresponding to the
+   AFGL (1986) atmospheric profiles
+   :cite:`Anderson1986AtmosphericConstituentProfiles`.
+   Stay tuned for that!
+
+.. admonition:: Example
+
+   Create a 1000 x 1000 x 120 km heterogeneous atmosphere with a radiative
+   properties profile approximately corresponding to the US76 atmosphere, using:
+
+   .. code:: python
+
+      import eradiate
+      from eradiate import unit_registry as ureg
+      eradiate.set_mode("mono")
+
+      atmosphere = eradiate.scenes.atmosphere.HeterogeneousAtmosphere(
+          width=ureg.Quantity(1000, "km"),
+          profile=dict(
+              type="us76_approx",
+              height=ureg.Quantity(120, "km"),
+              n_layers=50,
+          )
+      )
+
+   In this example, the atmosphere extends from 0 km to 120 km and is divided
+   into 50 layers.
+   In each of these layers, the albedo and the extinction coefficient are
+   automatically computed in the appropriate pressure and temperature conditions
+   corresponding to the US76 atmosphere, and at the current wavelength.
```

Comment:
This content should go to the tutorial (you will have an easier time maintaining the code if it's there).